src/sparse/KokkosSparse_sptrsv_supernode.hpp

```diff
@@ -1361,8 +1360,13 @@ invert_supernodal_columns(KernelHandle *kernelHandle, bool unit_diag, int nsuper
   using trmm_execution_space = typename KernelHandle::HandleExecSpace;
   using trmm_memory_space    = typename trmm_execution_space::memory_space;
   using trmm_view_t = Kokkos::View<scalar_t*, trmm_execution_space>;
+  #if defined(KOKKOS_ENABLE_CUDA) && !defined(KOKKOSKERNELS_ENABLE_TPL_CUBLAS)
+  // use KokkosBlas::trmm only with CUBLAS (since deep-copy to host throws an error)
```

Comment:
Is this a bug in trmm?

Reply:
I do not know if it is a bug or if it is meant to be. KokkosBlas::trmm deep-copies the matrices to host, and I think it does not support the matrix being a subviewe, throwing the exception.

Reply:
Ah, right. It is not a bug -- deep copy does not support copying of non-contiguous views between the host and device.